src/Module.Server/Modes/Dtv/CrpgDtvClient.cs

```diff
@@ -134,15 +137,16 @@ private void HandleViscountDeath(CrpgDtvGameEnd message)
     private void HandleViscountUnderAttack(CrpgDtvViscountUnderAttackMessage message)
     {
         var attackerAgent = Mission.MissionNetworkHelper.GetAgentFromIndex(message.AgentAttackerIndex, true);
+        var viscountAgent = Mission.MissionNetworkHelper.GetAgentFromIndex(message.AgentVictimIndex, true);
 
         if (attackerAgent == null)
```

Comment:
If we're going throught the hassle of making the code allow for any kind of agent to defend , then the code should be made "viscount agnostic" .
 The agent to defend can be called VIP or vip_to_defend.

Reply:
I have refactored references to 'viscount' in the code to VIP.

The check for "crpg_spawn_viscount" tag remains the same until a coordinated effort can be made to update scenes.

Reply:
> The check for "crpg_spawn_viscount" tag remains the same until a coordinated effort can be made to update scenes.

Spawn tag has been updated to `crpg_spawn_vip` and @rf-l has updated scenes accordingly.